### DISCUSSION TITLE
Add a 16 bit access code option to activating overrides

### DISCUSSION THREAD

**Discussion by MishraHarshvardhan428** [proginosko/LeechBlockNG#500] [Ideas]
RN the minimum character length is 32. And although I really appreciate how you have made it very difficult for procastinators like me to procastinate, if I wanna activate a 10 minute override it takes like 2 minutes or more. 16 would be the perfect number

**Comment by proginosko** [comment]
It's difficult to add this option due to the way the menu options are hard-coded. My advice would be to generate a random 16-character code (e.g., using a password generator) and enter it in the "Predefined access code" field.

**Reply by woj-tek** [reply]
Hmm... the problem is - at some point one starts to get better at entering this random string/code if it's the same ;)
